src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -37,7 +37,6 @@
 using System.Threading;
 using Microsoft.Identity.Client.ApiConfig;
```

Comment:
this ?

Reply:
AppConfig namespace is completely gone.  ApiConfig has two child namespaces which I'd prefer to keep.